e2e-tests/commands/insurance/complete-and-submit-name-on-policy-form.js

```diff
@@ -0,0 +1,27 @@
+import { FIELD_IDS } from '../../constants';
```

Comment:
Could we do `import INSURANCE_FIELD_IDS from '../../constants/field-ids/insurance';`, save some destructuring below?